keps/sig-api-machinery/1040-priority-and-fairness/README.md

```diff
@@ -2025,6 +2006,132 @@ Beta:
   - Automatically manages versions of mandatory/suggested configuration
   - Discrimates paginated LIST requests
 
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+* **How can this feature be enabled / disabled in a live cluster?**
+  - [x] Feature gate (also fill in values in `kep.yaml`)
+    - Feature gate name: APIPriorityAndFairness
+    - Components depending on the feature gate:
+      - kube-apiserver
+
+* **Does enabling the feature change any default behavior?** Yes, requests that
+  weren't rejected before could get rejected while requests that were rejected
+  previously may be allowed. Performance of kube-apiserver under heavy load
+  will likely be different too.
+
+* **Can the feature be disabled once it has been enabled (i.e. can we roll back
```

Comment:
Should we mention here that there is a command line flag that can be used to disable even when the FeatureGate is turned on?

Reply:
Added the flags to the first question, which tells the user that they can turn off any one of the three options (1 feature gate, 2 command line flags)